runtime.md

```diff
@@ -121,11 +121,18 @@ When the process in the container is stopped, irrespective of it being as a resu
 This operation MUST generate an error if it is not provided the container ID.
 Attempting to delete a container that does not exist MUST generate an error.
 Attempting to delete a container whose process is still running MUST generate an error.
-Deleting a container MUST delete the resources that were created during the `create` step.
-Note that resources associated with the container, but not created by this container, MUST NOT be deleted.
 Once a container is deleted its ID MAY be used by a subsequent container.
 
+Deleting a container MUST delete the resources that were created during the `create` step.
+Resources associated with the container, but not created by this container, MUST NOT be deleted.
+
+On Linux, [`mounts`](config.md#mounts), [`root`](config.md#root), [`linux.devices`](config-linux.md#devices), and other filesystem changes belong to the container which created the [container mount namespace][container-namespace3].
+If a container joins an existing mount namespace and applies those settings, deleting the container MUST NOT reverted the settings.
+Similarly [`hostname`](config.md#hostname) changes belong to the container which created the container UTS namespace.
+[`linux.resources`](config-linux.md#control-groups) settings belong to the container which created the associated cgroup.
```

Comment:
> I think the concept of "deleting resources owned only by the container" makes more sense…

I tried to cover that generically in the [preceding paragraph][1].  This paragraph gives some explicit examples to illustrate the concept and avoid ambiguity about who owns a given resource.  Do you feel that the preceding paragraph is sufficiently clear without these more-detailed examples?

[1]: https://github.com/opencontainers/runtime-spec/pull/651/commits/2e7155815d47dc415ae0b76a69a47584d7784ca7#diff-b84a8d65d8ed53f4794cd2db7e8ea731R126